frontend/src/metabase/query_builder/components/view/ViewHeader.jsx

```diff
@@ -284,6 +293,21 @@ export class ViewTitleHeader extends React.Component {
   }
 }
 
+function getExploreResultsUrl(question) {
+  return new Question(
+    {
+      dataset_query: {
+        database: question.query().database().id,
+        query: { "source-table": `card__${question.id()}` },
+        type: "query",
+      },
+      display: "table",
+      visualization_settings: {},
+    },
+    question.metadata(),
+  ).getUrl();
```

Comment:
Sorry for commenting after it's merged, but I think this could use [`composeThisQuery`](https://github.com/metabase/metabase/blob/master/frontend/src/metabase-lib/lib/Question.js#L513-L527).

Probably `question.composeThisQuery().setDisplay("table").setSettings({}).getUrl()`